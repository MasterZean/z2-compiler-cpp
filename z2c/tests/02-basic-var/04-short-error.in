val a = -32768s16
val b = 32768s16
val c = -40000s16
val d = 40000s16