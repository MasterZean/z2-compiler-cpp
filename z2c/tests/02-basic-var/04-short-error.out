int16 a = -32768;
DummyClass(2, 9): error:
	'Short' literal constant too big
DummyClass(3, 9): error:
	'Short' literal constant too big
DummyClass(4, 9): error:
	'Short' literal constant too big
